### ISSUE TITLE
Add android app platform version to specify/match a minSdk Version

### ISSUE THREAD

**Issue by Flocksserver** [snipsco/dinghy#85]
Hey,
on older android versions (< 7) I have the following error while loading a library created with cargo dinghy in my activity:

```kotlin
companion object {
        init {
            System.loadLibrary("my")
        }
    }
```

```
    java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "__gnu_Unwind_Find_exidx" referenced by "/data/app/<package>/lib/arm/libmy.so"...
```
"__gnu_Unwind_Find_exidx"
"__register_atfork"
or other "symbols" on different devices 

Newer Android versions do not have this problem. I guess that is a problem with the APP_PLATFORM variable for android ndk builds? This version should be not lower than the android os version (api version)?

Is there a possible way to specify this APP_PLATFORM Version?

Best regards :)

**Comment by kali** [comment]
@Deluvi is this covered by your changes, or is it something else ? (cc @fredszaq )

**Comment by fredszaq** [comment]
Are we talking of android api 7 (eclair 2.1, released circa 2009, not supported in the ndk since 2016) or android 7.0 (nouggat api 24, release in 2016) ?

@Deluvi 's [PR](https://github.com/snipsco/dinghy/pull/83) should fix any problems regarding android 7.0, just use the new `auto-android-[ARCHITECTURE]-api[API_LEVEL]` platform on ndk 19 and you should be good (available in dinghy 0.4.10)

**Comment by Flocksserver** [comment]
> [PR](https://github.com/snipsco/dinghy/pull/83) should fix any problems regarding android 7.0, just use the new `auto-android-[ARCHITECTURE]-api[API_LEVEL]` platform on ndk 19 and you should be good (available in dinghy 0.4.10)

yesterday! NICE ❤️ @Deluvi that is exactly what I was looking for!

with an update on dinghy 0.4.10 it works:

```
cd cargo && cargo dinghy --platform auto-android-armv7-api19 build --release
 INFO  cargo_dinghy > Targeting platform 'auto-android-armv7-api19' and device '<none>'
    Finished release [optimized] target(s) in 0.38s  
``` 
but the following:

```
cd cargo && cargo dinghy --platform auto-android-aarch64-api19 build
 ERROR cargo_dinghy > Error: No 'auto-android-aarch64-api19' platform found

make: *** [build-lib-android-aarch64-debug] Error 1

```

**Comment by fredszaq** [comment]
that's completely normal, 64bit platforms are not supported on android before api 21
